### PR TITLE
Updating GBP values: The Re-GBP-ening

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
I'm satisfied with the change of pace we've noticed with the repo as of late, and as such, I'm going back on some values of GBP, namely lowering fixes by one and moving Mechanics (read: features) back down to -10, from -15. Yes, I clicked the wrong button on accident and sent the PR before I was done writing.

I also wanted PR 10000.

## How This Contributes To The Skyrat Roleplay Experience
Being less harsh on people that put `add` on everything they make.

### CODE REVIEW

**Comment by tf-4** [comment]
Considering the array of performance issues present on the server, I'm not sure we should be shying away from encouraging fixes in favour of features.

**Comment by softcerv** [comment]
>Considering the array of performance issues present on the server, I'm not sure we should be shying away from encouraging fixes in favour of features.

I'm fine with the feature cost being lowered, but at the same time i don't think the points gain from fixes should be decreased

**Comment by softcerv** [comment]
also happy 10000th PR

File: .github/gbp.toml [hunk 1]
```diff
@@ -8,11 +8,11 @@ reset_label = "GBP: Reset"
 "Code Improvement" = 2
 "Content Expansion" = -3
 "Feedback" = 2
-"Fix" = 4
+"Fix" = 3
```
Comment (GoldenAlpharex) [inline .github/gbp.toml]:
```suggestion
"Fix" = 4
```

File: .github/gbp.toml [hunk 2]
```diff
@@ -8,11 +8,11 @@ reset_label = "GBP: Reset"
...
 "Grammar and Formatting" = 1
 "Hard Deletes" = 12
 "Logging" = 1
-"Mechanic" = -15
+"Mechanic" = -10
```
Comment (GoldenAlpharex) [inline .github/gbp.toml]:
```suggestion
"Mechanic" = -12
```

**Comment by ChungusGamer666** [comment]
Remove GBP entirely. It just keeps people away from developing features and makes the code worse as a result.
I'm sorry but telling someone "no, not enough good boy points, your feature is bye bye" will not make them go and fix bugs, it'll just make them fuck off and code for another server.

**Comment by GoldenAlpharex** [comment]
> Remove GBP entirely. It just keeps people away from developing features and makes the code worse as a result. I'm sorry but telling someone "no, not enough good boy points, your feature is bye bye" will not make them go and fix bugs, it'll just make them fuck off and code for another server.

It's funny because I have said that maybe once since we started enforcing it, and a lot more fixes have been happening. Can't argue with the results.

**Pull request merged**